Scripts/Services/CompareComply/V1/CompareComplyService.cs

```diff
@@ -1,5 +1,5 @@
 /**
-* (C) Copyright IBM Corp. 2018, 2020.
+* Copyright 2018, 2019 IBM Corp. All Rights Reserved.
```

Comment:
was lint-copyrights run on this file? Looks like the old copyright

Reply:
ooh, I ran it on all of them, let me see if this one was missed somehow.